docs/howtos/policy-servers/03-production-deployments.md

```diff
@@ -0,0 +1,209 @@
+---
+sidebar_label: Production deployments
+title: Configuring PolicyServers for production
+description: Configuring PolicyServers for production
+keywords:
+  [
+    kubewarden,
+    kubernetes,
+    policyservers,
+    production,
+    poddisruptionbudget,
+    affinity,
+    limits,
+  ]
+doc-persona: [kubewarden-operator, kubewarden-integrator]
+doc-type: [howto]
+doc-topic:
+  [
+    operator-manual,
+    policy-servers,
+    production,
+    poddisruptionbudget,
+    affinity,
+    limits,
+  ]
+---
+
+PolicyServers are critical to the cluster. Reliability of them is important as
```

Comment:
We should have some guidelines about how to spell certain pieces of our tech stack. Do we have to use `PolicyServer` or `Policy Server`?

It would be even better to have some keywords that automatically expand to the right term. For example, `&policyServer` gets expanded to be `PolicyServer` at rendering time. I don't know if docusaurus has something for that

CC @jhkrug

Reply:
I think we have agreed on PolicyServer, where talking about the component in general. As per the Kubernetes docs style guide. And yes, I can do something about variables in DS. I had to do it for Longhorn. It would be something like

`[[< policy-server-component >]] expands to `PolicyServer`, etc.

Reply:
Opened https://github.com/kubewarden/docs/issues/394 to not forget about it.